CDP4PluginPackager.Tests/AppTestFixture.cs

```diff
@@ -2,7 +2,7 @@
 // <copyright file="AppTestFixture.cs" company="RHEA System S.A.">
 //    Copyright (c) 2015-2020 RHEA System S.A.
 //
-//    Author: Sam Geren�, Alex Vorobiev, Merlin Bieze, Naron Phou, Patxi Ozkoidi, Alexander van Delft,
+//    Author: Sam Geren�, Alex Vorobiev, Merlin Bieze, Naron Phou, Patxi Ozkoidi, Alexander van Delft,
```

Comment:
revert pls, seems you have encodeing issue maybe? Check that you are correctly set to UTF-8.